lxd/instance/drivers/driver_qemu.go

```diff
@@ -5543,31 +5543,38 @@ func (d *qemu) writeInstanceData() error {
 // Info returns "qemu" and the currently loaded qemu version.
 func (d *qemu) Info() instance.Info {
 	data := instance.Info{
-		Name: "qemu",
+		Name:  "qemu",
+		Error: fmt.Errorf("Unknown error"),
 	}
 
 	hostArch, err := osarch.ArchitectureGetLocalID()
 	if err != nil {
+		data.Error = fmt.Errorf("Failed getting architecture")
 		return data
 	}
 
 	qemuPath, _, err := d.qemuArchConfig(hostArch)
 	if err != nil {
+		data.Error = fmt.Errorf("QEMU command not available for architecture")
 		return data
 	}
 
 	out, err := exec.Command(qemuPath, "--version").Output()
 	if err != nil {
+		data.Error = fmt.Errorf("Failed getting QEMU version")
 		return data
 	}
 
 	qemuOutput := strings.Fields(string(out))
 	if len(qemuOutput) < 4 {
-		data.Version = "unknown"
+		data.Version = "unknown" // Not necessarily an error that should prevent us using driver.
```

Comment:
But it will still be treated as such since you initialize `Error` with `Unknown error` at the beginning.
So if an unknown version should be accepted, this needs to do `data.Error = nil`

Reply:
Oh yes good spot!